Gopkg.toml

```diff
@@ -16,7 +16,7 @@
 
 [[constraint]]
   name = "github.com/aws/aws-sdk-go"
-  version = "1.15.54"
+  version = "1.19.41"
```

Comment:
Undo this change, since we don't really require a newer version than this to build.  When you open a pull request that requires the newer version you can then set it to, ideally, the first version that would support the feature you need.

The idea is to allow builders flexibility in the version they use in case they decide not to use our lock values, which is nice for system integrators.

Reply:
Aside from the new input we're working on, the more immediate need for the new version of the aws SDK is to pull some new ELB metrics for the existing CloudWatch input. This does not require a change to the CloudWatch telegraf input, does require a newer version of the AWS SDK.

Reply:
Updated the PR description to focus more on this particular issue.

Reply:
Can you tell me a bit more about this?  What is you want to do and what feature of the current sdk is missing?

Reply:
Which version is required for the SpilloverCount metric to work?